remote-location-assert.md

```diff
@@ -0,0 +1,88 @@
+
+# Stakeholders
+[stakeholders]: #stakeholders
+
+Any Helium hotspot owner who has deployed at least one remote hotspot. The issue of not being able to assert location remotely has caused a large outcry within the hotspot owner community on Slack and Telegram.
+
+The Helium blockchain engineering team.
+
+# Detailed Explanation
+[detailed-explanation]: #detailed-explanation
+
+## Implement a new transaction, `assert_location_v2`, and optionally deprecate the previous `assert_location_v1`
+
+This new transaction would require the owner's private key, hotspot public key and lat/long coords of the new desired location. It would be submitted to the blockchain to replace the hotspot's existing location, and charge the same assert fee (in DC) as `assert_location_v1`.
+
+## Create a simple UI for asserting hotspot location in the Helium mobile app
+
+This UI would be the same as the current pin-drop UI, centering the map on either the hotspot's current location, or the phone's current location if the hotspot has no location.
+
+It would need to be easy to find. It could be exposed as a new button next to the gear-icon button that is currently used to bring up the Bluetooth connection UI, or possibly as a separate section on the Bluetooth connection modal, where it's made clear that a direct connection to the hotspot isn't required to use this function.
+
+If the hotspot's current asserted location is far (>1km) from the user's current GPS location, they should be warned that they are changing the location for a remote hotspot and picking the wrong location could have an impact on that hotspot's earnings.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+It will add a new type of transaction which will have to be maintained and considered in all parts of the codebase where the current `assert_location_v1` transaction is considered.
+
+# Rationale and Alternatives
+[alternatives]: #rationale-and-alternatives
+
+This is a simple and straightforwad solution that will save hotspot owners time, effort and frustration. It will reduce the amount of chatter and complaints clogging up the chat channels about how frustrating it is to not be able to do this.
+
+Not making this change could deter current and future owners from making remote deployments, which could be viewed as too much of an operational hassle. This could hurt the growth of the network.
+
+Hotspot owners could be upset if they aren't able to update a far-away hotspot before the GPS assertion check is enforced and they lose their earnings until they can access it.
+
+It will generally reduce operational friction, and friction in the community.
+
+# Unresolved Questions
+[unresolved]: #unresolved-questions
+
+Are there any security, privacy or other implications of allowing location to be remotely asserted with only the account holder's private key and not the hotspot's signature?
```

Comment:
Not having the hotspot's signature in `assert_location_v2` transaction should be fine.